source/leuven/genius_quick_start.rst

```diff
@@ -94,8 +104,24 @@ To specifically request V100 GPUs (which are on nodes with CascadeLake architect
   
 For the V100 type of GPU, it is required that you request 4 cores for each GPU. Also notice that these nodes offer much larger memory bank.
 
-Advanced usage
-^^^^^^^^^^^^^^
+Advanced node usage
+^^^^^^^^^^^^^^^^^^^
+The node access policy on Skylake and AMD nodes is `SINGLEUSER`.
```

Comment:
```suggestion
The node access policy on Skylake nodes is `SINGLEUSER`.
```

(I still think it would be confusing to mention the AMD nodes in this section
about the "batch" partition. In case you want to tell something about the  AMD
nodes, I think it's best to put that in the "Submit to an AMD node" section.)